dateparser/date_parser.py

```diff
@@ -170,7 +170,7 @@ def dateutil_parse(date_string, **kwargs):
 
 
 class DateParser(object):
-    def parse(self, date_string):
+    def parse(self, date_string, dayfirst=False, yearfirst=False):
```

Comment:
Hm, maybe it's better to use `**kwargs` here. I mean we just passing these arguments to `dateutil_parse` function.